pkg/agentfs/build.go

```diff
@@ -0,0 +1,114 @@
+package agentfs
```

Comment:
what does `fs` stand for?

Reply:
agent-filesystem.. was trying to be a bit generic since there's a lot of functionality in this package.